podwatcher/podwatcher_test.go

```diff
@@ -124,7 +124,7 @@ var _ = Describe("podwatcher", func() {
 			})
 
 			It("Doesn't add any containers if the guid is empty", func() {
-				delete(pod.ObjectMeta.Labels, "guid")
+				delete(pod.ObjectMeta.Labels, "cloudfoundry.org/guid")
```

Comment:
Better to user the constant here instead of copying
https://github.com/cloudfoundry-incubator/eirini/blob/ca5617920508538c055485a30362b4240b2d35ee/k8s/statefulset.go#L38